### ISSUE TITLE
New work-in-progress menu.

### ISSUE THREAD

**Issue by lodici** [magarena/magarena#1117]
I have added a work-progress-menu which is accessible from the devmode menu. I have so many experimental branches in my github repository that I lose track of what I have done so I thought it would be useful to have an area in Magarena that can be used to shared bits & pieces of code that are not quite finished but might have some future use or be of interest to someone.

**Comment by lodici** [comment]
![screenshot00605](https://cloud.githubusercontent.com/assets/2443519/25772350/cbc69a8e-3260-11e7-80b7-15042149ee56.jpg)

This cardflow code has been hanging around in my repository for ages but I never could quite decide how or where it might be included and actually it feels like it is pushing the limits of Swing with the graphical animation required but the end effect is quite cool. As well as the navigation buttons shown in the screenshot, you can also scroll back and forth through the cards using the arrow keys or mouse wheel.

**Comment by ShawnieBoy** [comment]
Oooh, I like that - I can think of a couple of places it could be used:

1st hand mulligan options
fromCardList search e.g. Searching library, graveyard etc.

Even a 'New Cards added' viewer

**Comment by lodici** [comment]
I updated the Cardflow screen and now display it when clicking on a card in the library screen (only available in devmode or when running a test game). It now shows the top card using high quality scaling but I found that high quality scaling of all images seriously affected the animation. I also updated it so that the actual number of cards has no affect on performance or resources - it only ever needs to draw about 12 images at a time and only requests new images when required. I tested it with 200 cards and it was fine. I have also speeded up the scrolling/animation.
